tests/test_core/test_alignment.py

```diff
@@ -1933,6 +1933,15 @@ def test_copy(self):
         copied = aln.copy()
         self.assertEqual(aln.info, copied.info)
 
+    def test_deep_copy(self):
```

Comment:
make this a method if `AlignmentTests`.

Plus, add tests for `just_span=False`